BookingQueueSubscriber/BookingQueueSubscriber.Services/NotificationApi/AddNotificationRequestMapper.cs

```diff
@@ -238,7 +239,10 @@ public static AddNotificationRequest MapToDemoOrTestNotification(HearingDto hear
                 else
                 {
                     notificationType = NotificationType.JudgeDemoOrTest;
-                    contactEmail = contactEmailForNonEJudJudgeUser ?? participant.ContactEmail;
+                    if (!string.IsNullOrEmpty(contactEmailForNonEJudJudgeUser))
```

Comment:
As per the SonarCloud suggestions, this if statement is redundant as it will always evaluate to true, so should be able to remove that check and just have

`contactEmail = contactEmailForNonEJudJudgeUser;`

Reply:
removed the if statement